src/Cache_Command.php

```diff
@@ -155,8 +155,12 @@ public function delete( $args, $assoc_args ) {
 	 *     Success: The cache was flushed.
 	 */
 	public function flush( $args, $assoc_args ) {
-		$value = wp_cache_flush();
 
+		if ( WP_CLI::has_config( 'url' ) && ! empty( WP_CLI::get_config()['url'] ) && is_multisite() ) {
+			WP_CLI::warning( 'You are using a multisite installation. Flushing the cache will flush the cache for all sites.' );
```

Comment:
What do you think about a message like this?

> Ignoring the `--url=<url>` argument because flushing the cache affects all sites on a multisite installation.

I think it's worth explicitly mentioning the cause of the warning message. I also don't know how often we use "You" in messages...

<img width="645" alt="image" src="https://user-images.githubusercontent.com/36432/230089932-c92e5db5-f7bb-470e-b1f7-fc4387769519.png">